sdcm/tester.py

```diff
@@ -1562,7 +1562,7 @@ def get_cluster_k8s_eks(self):  # pylint: disable=too-many-statements
         self.k8s_cluster.deploy_node_pool(
             eks.EksNodePool(
                 name=self.k8s_cluster.AUXILIARY_POOL_NAME,
-                num_nodes=2,
+                num_nodes=3,
```

Comment:
I guess we should have configuration for this, since in non serverless runs, we don't need this extra node.

Reply:
There's more about it  - I suppose in larger deployments we'll need to scale it out/up. I'll be verifying it in many-clients-test (and if needed I'll fix it).

Reply:
I agree that we should have a config option for it if we want to have more than 1 value here.

Reply:
create a configuration for it